vmngclient/utils/dashboard.py

```diff
@@ -34,16 +73,113 @@ def name_converter(name):
 
 @define
 class Count(DataclassBase):
-    name: Union[DeviceName, StatusName] = field(converter=name_converter)
-    count: int
     details_url: str = field(metadata={FIELD_NAME: "detailsURL"})
-    status: str
+    count: Optional[int] = field(default=None)
+    status: Optional[str] = field(default=None)
+    name: Optional[
+        Union[DeviceName, StatusName, BfdConnectivityName, InventoryName, PercentageDistributionName]
+    ] = field(default=None, converter=name_converter)
     message: Optional[str] = field(default=None)
-    status_list: Union[Optional["Count"]] = field(default=None, metadata={FIELD_NAME: "statusList"})
+    status_list: Optional[List["Count"]] = field(default=None, metadata={FIELD_NAME: "statusList"})
+    list: Optional[str] = field(default=None)
+    value: Optional[int] = field(default=None)
+    percentageDistribution: Optional[PercentageDistribution] = field(default=None)
+
+    @value.validator  # type: ignore
+    def value_or_count_provided(self, attribute, value):
+        if value is None and self.count is None:
+            raise ValueError("For 'Count' dataclass 'value' or 'count' attribute must be provided.")
 
 
 @define
 class CertificatesStatus(DataclassBase):
     invalid: int
     warning: int
     revoked: int
+
+
+@define
+class ControlStatus(DataclassBase):
+    up: int = field(metadata={FIELD_NAME: "controlUp"})
+    partial: int
+    down: int = field(metadata={FIELD_NAME: "controlDown"})
+
+
+@define
+class SiteHealth(DataclassBase):
+    full_connectivity: int = field(metadata={FIELD_NAME: "fullConnectivity"})
+    partial_connectivity: int = field(metadata={FIELD_NAME: "partialConnectivity"})
+    no_connectivity: int = field(metadata={FIELD_NAME: "noConnectivity"})
+
+
+@define
+class vEdgeHealth(DataclassBase):
+    normal: int
+    warning: int
+    error: int
+
+
+@define
+class vSmartStatus(DataclassBase):
+    up: int
+    down: int
```

Comment:
does 0 mean false and 1 true? Can we have docstring here

Reply:
same as in vedges - number of vsmarts with status

Reply:
The integer indicates how many vSmarts have given status, there is no true/false interpretation